build.gradle.kts

```diff
@@ -1,31 +1,32 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
-buildscript {
+plugins {
```

Comment:
I'm out of my depth here. Would you have a link to some nice doc that explains that in more details? Thanks!

Reply:
https://docs.gradle.org/current/userguide/plugins.html#sec:plugins_block 

Also this is often referred as Plugin DSL or the `plugins{}` block.